moab_action_transform.ink

```diff
@@ -0,0 +1,132 @@
+###
+
+# MSFT Bonsai 
+# Copyright 2020 Microsoft
+# This code is licensed under MIT license (see LICENSE for details)
+
+# Moab Sample Using Action Transform during Training Only
+
+###
+
+inkling "2.0"
+
+using Math
+using Goal
+
+# Distances measured in meters
+const RadiusOfPlate = 0.1125 # m
+
+# Velocities measured in meters per sec.
+const MaxVelocity = 6.0
+const MaxInitialVelocity = 1.0
+
+# Threshold for ball placement
+const CloseEnough = 0.02
+
+# Default time delta between simulation steps (s)
+const DefaultTimeDelta = 0.045
+
+# Maximum distance per step in meters
+const MaxDistancePerStep = DefaultTimeDelta * MaxVelocity
+
+# Maximum plate angle in degrees
+const MaxPlateAngle = 22
+
+# State received from the simulator after each iteration
+type ObservableState {
+    # Ball X,Y position
+    ball_x: number<-MaxDistancePerStep - RadiusOfPlate .. RadiusOfPlate + MaxDistancePerStep>,
+    ball_y: number<-MaxDistancePerStep - RadiusOfPlate .. RadiusOfPlate + MaxDistancePerStep>,
+
+    # Ball X,Y velocity
+    ball_vel_x: number<-MaxVelocity .. MaxVelocity>,
+    ball_vel_y: number<-MaxVelocity .. MaxVelocity>,
+}
+
+type BrainAction {
+    input_pitch: number<-MaxPlateAngle .. MaxPlateAngle>, # rotate about x-axis
+    input_roll: number<-MaxPlateAngle .. MaxPlateAngle>, # rotate about y-axis
+}
+
+# Action provided as output by policy and sent as
+# input to the simulator
+type SimAction {
+    # Range -1 to 1 is a scaled value that represents
+    # the full plate rotation range supported by the hardware.
+    input_pitch: number<-1 .. 1>, # rotate about x-axis
+    input_roll: number<-1 .. 1>, # rotate about y-axis
+}
+
+# Per-episode configuration that can be sent to the simulator.
+# All iterations within an episode will use the same configuration.
```

Comment:
Unrelated to your PR, but: per recent discussion in Teams, we should delete this line — it's confusing since we're only configuring initial conditions, not constants like ball mass.

Reply:
deleting `# All iterations within an episode will use the same configuration.`